docs/setup/system-requirements.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: "System Requirements"
+sidebarTitle: "System Requirements"
+---
+
+## System Requirements for Odigos
+To ensure optimal performance and compatibility when deploying Odigos, your environment must meet the following system requirements:
+
+**Kubernetes Version**
+Minimum Version: Odigos requires Kubernetes version 1.20.3.0 or higher.
+
+**Operating System**
+Supported Platforms: Odigos operates on systems with an underlying Linux kernel version of 5.4.0 or higher.
```

Comment:
The linux kernel version is only relevant to eBPF instrumentations (only golang for opensource).

All "natvie" agents can run on any linux nodes

I think it's worth mentioning